com.sap.cloud.lm.sl.cf.api/src/main/java/com/sap/cloud/lm/sl/cf/web/api/model/State.java

```diff
@@ -12,11 +12,15 @@
 public enum State {
```

Comment:
Can we simplify this enum further?
Its elements use an XmlEnumValue annotation, using the same value as the enum's name.
This is in the doc of XmlEnumValue:
"In the absence of this annotation, Enum.name() is used as the XML representation. "
Another point is the 'value' variable inside the enum, which also uses the same. Can these two be completely removed to simplify the enum?
Check the usages in the code to verify this.